test/test_other.py

```diff
@@ -14018,3 +14018,18 @@ def test_hello_world_argv(self):
 
   def test_arguments_global(self):
     self.emcc(test_file('hello_world_argv.c'), ['-sENVIRONMENT=web', '-sSTRICT', '--closure=1', '-O2'])
+
+  @parameterized({
+    'no_std_exp':   (['-DEMMALLOC_NO_STD_EXPORTS'],),
+    # When we let emmalloc build with the standard exports like malloc,
+    # emmalloc == malloc.
+    'with_std_exp': (['-DTEST_EMMALLOC_IS_MALLOC'],),
+  })
+  def test_emmalloc_in_addition(self, args):
+    # Test that we can use emmalloc in addition to another malloc impl. When we
+    # build emmalloc using -DEMMALLOC_NO_STD_EXPORTS it will not export malloc
+    # etc., and only provide the emmalloc_malloc etc. family of functions that
+    # we can use.
+    emmalloc = path_from_root('system', 'lib', 'emmalloc.c')
```

Comment:
When you build emmalloc in addition to malloc here.. how does it work?    How does emmalloc get it underlying memory?  Are they both able to grab memory from the sbrk region somehow?

Reply:
Both use sbrk to allocate. Neither frees anything to sbrk, so things work out.

(This won't be an issue for the intended use of this, though, which is that the second allocator calls emmalloc which calls sbrk.)